### ISSUE TITLE
Graphics card switching doesn't show on 19.04

### ISSUE THREAD

**Issue by trgeiger** [pop-os/gnome-shell-extension-system76-power#20]
**Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="19.04"
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
Installed: 1.0.0~1553533661~19.04~8a5af31

**Issue/Bug Description:**
I manually upgraded to 19.04 early to help pick out bugs and to try out Gnome 3.32. Looks like the graphics card switching option doesn't show up in the tray after the update.

**Steps to reproduce (if you know):**
Click the battery dropdown in the Gnome panel and there's no longer any options for Nvidia/Intel. The CPU options do still show.

**Expected behavior:**
Should show Nvidia/Intel options.

**Other Notes:**
Using system76-power on the command line still works--I can switch between the graphics card switch no problem. It seems it's just an issue with the Gnome extension, as far as I can tell.

**Comment by brs17** [comment]
I can confirm this issue on a oryp5. @trgeiger , you are right, it does seem to be an issue with the gnome shell extension.

Logs:
```
Apr 08 10:17:52 pop-os gnome-shell[1672]: Some code called array.toString() on a Uint8Array instance. Previously this would have interpreted the bytes of the array as a string, but that is n
                                          (Note that array.toString() may have been called implicitly.)
                                          0 init() ["/usr/share/gnome-shell/extensions/system76-power@system76.com/extension.js":57]
                                          1 initExtension() ["resource:///org/gnome/shell/ui/extensionSystem.js":245]
                                          2 loadExtension() ["resource:///org/gnome/shell/ui/extensionSystem.js":174]
                                          3 _loadExtensions/<() ["resource:///org/gnome/shell/ui/extensionSystem.js":324]
                                          4 _emit() ["resource:///org/gnome/gjs/modules/signals.js":128]
                                          5 _loadExtension() ["resource:///org/gnome/shell/misc/extensionUtils.js":251]
                                          6 scanExtensions/<() ["resource:///org/gnome/shell/misc/extensionUtils.js":257]
                                          7 collectFromDatadirs() ["resource:///org/gnome/shell/misc/fileUtils.js":25]
                                          8 scanExtensions() ["resource:///org/gnome/shell/misc/extensionUtils.js":256]
                                          9 _loadExtensions() ["resource:///org/gnome/shell/ui/extensionSystem.js":326]
                                          10 enableAllExtensions() ["resource:///org/gnome/shell/ui/extensionSystem.js":334]
                                          11 _sessionUpdated() ["resource:///org/gnome/shell/ui/extensionSystem.js":365]
                                          12 init() ["resource:///org/gnome/shell/ui/extensionSystem.js":373]
                                          13 _initializeUI() ["resource:///org/gnome/shell/ui/main.js":225]
                                          14 start() ["resource:///org/gnome/shell/ui/main.js":127]
                                          15 <TOP LEVEL> ["<main>":1]
Apr 08 10:17:52 pop-os gnome-shell[1672]: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.system76.PowerDaemon was not provided by any .service files
Apr 08 10:17:52 pop-os gsd-media-keys[1783]: g_variant_get_va: assertion 'value != NULL' failed
Apr 08 10:17:52 pop-os gsd-media-keys[1783]: g_variant_unref: assertion 'value != NULL' failed
Apr 08 10:17:52 pop-os gnome-shell[1672]: Extension "system76-power@system76.com" had error: Gio.DBusError: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name com.system76.Power
Apr 08 10:17:52 pop-os gnome-shell[1672]: JS ERROR: Could not load extension ubuntu-dock@ubuntu.com: Error: Missing metadata.json
                                          createExtensionObject@resource:///org/gnome/shell/misc/extensionUtils.js:170:15
                                          _loadExtension@resource:///org/gnome/shell/misc/extensionUtils.js:246:25
                                          scanExtensions/<@resource:///org/gnome/shell/misc/extensionUtils.js:257:13
                                          collectFromDatadirs@resource:///org/gnome/shell/misc/fileUtils.js:25:17
                                          scanExtensions@resource:///org/gnome/shell/misc/extensionUtils.js:256:9
                                          _loadExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:326:5
                                          enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:334:9
                                          _sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:365:9
                                          init@resource:///org/gnome/shell/ui/extensionSystem.js:373:5
                                          _initializeUI@resource:///org/gnome/shell/ui/main.js:225:5
                                          start@resource:///org/gnome/shell/ui/main.js:127:5
                                          @<main>:1:31

```

And for reference, this is what the extension looks like in the broken state:
![Screenshot from 2019-04-08 10-21-51](https://user-images.githubusercontent.com/7460774/55740860-38666f00-59e9-11e9-924c-7bddf7b9d2d6.png)

**Comment by mmstick** [comment]
> The name com.system76.PowerDaemon was not provided by any .service files

This would indicate that the power daemon is not running (?).

**Comment by Luke-Nukem** [comment]
Same issue here, can confirm the daemon is receiving events though.
![Uploading Screenshot from 2019-04-09 17-52-57.png…]()

**Comment by trgeiger** [comment]
I also checked yesterday--system76-power is indeed running according to systemctl.

**Comment by jackpot51** [comment]
Fixed by 772708e21138990f79ca343cc02a50c28b19b4b6
